app/api/auth/logout/route.ts

```diff
@@ -0,0 +1,21 @@
+import { AxiosError } from 'axios';
+import { NextResponse } from 'next/server';
+import api from '@/api/serverSideAxiosConfig';
+
+export async function POST() {
```

Comment:
Does the name "POST" make sense here?

Reply:
Es un route handler de NextJs (https://nextjs.org/docs/app/building-your-application/routing/router-handlers)

Reply:
O decis que sea un GET?